### PR TITLE
Changed indentation size from 2 to 4

### PR DESCRIPTION
**Pull request by Frodigo**
In PWA Studio there is a .editorconfig file where you can find configuration for indentation: 
Zrzut ekranu 2020-10-28 o 08 38 49

In Prettier, default indentation is 2, and this causes problems. 

Because of that, I would like to change the indentation size from 2 to 4 and be consistent with PWA Studio.

### CODE REVIEW

**Review by larsroettig — APPROVED** [review]

File: packages/cpse-template/prettier.config.js [hunk 1]
```diff
@@ -1,5 +1,6 @@
 const config = {
-  singleQuote: true
+    singleQuote: true,
```
Comment (larsroettig) [inline packages/cpse-template/prettier.config.js:2]:
```
const config = {
    singleQuote: true,
    tabWidth: 4,
    trailingComma: 'none'
};
```

**Review by larsroettig — COMMENTED** [review]
Can you also add them `.editorconfig` in root ?

```
root = true

[*]
charset = utf-8
end_of_line = lf
indent_size = 4
indent_style = space
insert_final_newline = true
trim_trailing_whitespace = true

[{package.json,*.yml,*.md}]
indent_size = 2
indent_style = space

[*.md]
trim_trailing_whitespace = false
```

File: packages/cpse-template/prettier.config.js (hunk 1 above)
Comment (Frodigo) [reply packages/cpse-template/prettier.config.js:2]: Changed!

**Comment by Frodigo** [comment]
Hi Lars, I added the .editocgonfig file, also I updated the Prettier config.

**Review by larsroettig — APPROVED** [review]

**Pull request merged**